eng/Signing.props

```diff
@@ -6,6 +6,10 @@
     <ItemsToSign Include="$(BinDir)*.exe" />
   </ItemGroup>
 
+  <ItemGroup>
+    <FileSignInfo Include="mscordaccore.dll" CertificateName="MicrosoftSHA2" />
+  </ItemGroup>
+
```

Comment:
Does this run before or after the long-named dac is created?  It would be good if both DLLs receive the same certificates, since they are supposed to be exact copies of each other.